dev/com.ibm.ws.microprofile.metrics.2.3/src/com/ibm/ws/microprofile/metrics23/writer/PrometheusMetricWriter23.java

```diff
@@ -66,10 +70,7 @@ protected void writeMetricMapAsPrometheus(StringBuilder builder, String registry
             double conversionFactor = conversionAppendEntry.getValue();
             String appendUnit = conversionAppendEntry.getKey();
 
-            //If current metadata that we are parsing does not have a matching metric... skip
-            if (currentMetricMap.isEmpty()) {
-                continue;
-            } else if (metricMetadata.getTypeRaw().equals(MetricType.COUNTER)) {
+            //if (metricMetadata.getTypeRaw().equals(MetricType.COUNTER)) {
```

Comment:
Uncomment the if statement

Reply:
ah right, may have pushed that commit a little bit late in the day :D